### PR TITLE
Upgrade 0.64

### LINKED ISSUE #38: Update gallery to RNW 0.64 release
RNW is currently previewing the 0.64 release. As of right now the 5th preview us up:
- [react-native-windows_v0.64.0-preview.5](https://github.com/microsoft/react-native-windows/releases/tag/react-native-windows_v0.64.0-preview.5)

It would be useful to validate the 0.64 previews with this gallery app (see issue #23). Let's get a build of gallery on 0.64 ready so we can start testing it.

### PR DESCRIPTION
**Pull request by chiaramooney**
Upgrade gallery to new release of RNW: v0.64 

Resolves #38

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,31 +10,31 @@
...
+    "@react-native-picker/picker": "^1.9.12",
     "@react-navigation/bottom-tabs": "^5.8.0",
     "@react-navigation/drawer": "^5.9.0",
     "@react-navigation/native": "^5.7.3",
     "@react-navigation/stack": "^5.9.0",
     "@terrylinla/react-native-sketch-canvas": "creambyemute/react-native-sketch-canvas",
     "lowlight": "^1.17.0",
     "react": "16.13.1",
-    "react-native": "0.63.2",
+    "react-native": "^0.64.0-rc.1",
```
Comment (chrisglein) [inline package.json:26]: Should this not have the "rc" on it?

File: windows/rngallery/rngallery.vcxproj [hunk 2]
```diff
@@ -15,6 +16,7 @@
     <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
     <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.18362.0</WindowsTargetPlatformVersion>
     <WindowsTargetPlatformMinVersion>10.0.16299.0</WindowsTargetPlatformMinVersion>
+    <PackageCertificatePassword>password</PackageCertificatePassword>
```
Comment (chrisglein) [inline windows/rngallery/rngallery.vcxproj:19]: What's this? I assume it has to do with the new pfx process, but maybe there needs to be a wiki/md update to explain this? Or a comment inline here?

File: package.json (hunk 1 above)
Comment (chiaramooney) [reply package.json:26]: I updated the gallery using `npx react-native-windows-init --version 0.64.0 --overwrite` which automatically selects a copy of React Native to install for you. My guess is that it just selects the most up to date release that's compatible?

File: windows/rngallery/rngallery.vcxproj (hunk 2 above)
Comment (chiaramooney) [reply windows/rngallery/rngallery.vcxproj:19]: That line was also generated by `npx react-native-windows-init --version 0.xx-preview.x --overwrite` I believe. Let me test if I can remove it and see if the build still succeeds.

Comment (chiaramooney) [reply windows/rngallery/rngallery.vcxproj]: Ok just tested. Looks like it isn't needed.

**Review by kmelmon — APPROVED** [review]
:shipit:

**Pull request merged**